measures/2023/measures-data.json

```diff
@@ -1290,18 +1290,22 @@
     "category": "pi",
     "measureId": "PI_EP_2",
     "title": "Query of the Prescription Drug Monitoring Program (PDMP)",
-    "description": "For at least one Schedule II opioid electronically prescribed using CEHRT during the performance period, the MIPS eligible clinician uses data from CEHRT to conduct a query of a PDMP for prescription drug history, except where prohibited and in accordance with applicable law.",
-    "isRequired": false,
+    "description": "For at least one Schedule II opioid or Schedule III or IV drug electronically prescribed using CEHRT during the performance period, the MIPS eligible clinician uses data from CEHRT to conduct a query of a PDMP for prescription drug history.",
+    "isRequired": true,
     "metricType": "boolean",
     "firstPerformanceYear": 2019,
     "lastPerformanceYear": null,
     "objective": "electronicPrescribing",
-    "isBonus": true,
-    "reportingCategory": "bonus",
+    "isBonus": false,
+    "reportingCategory": "required",
     "substitutes": [],
     "measureSpecification": {},
     "measureSets": [],
-    "exclusion": null
+    "exclusion": [
+      "PI_EP_2_EX_1",
+      "PI_EP_2_EX_2",
+      "PI_EP_2_EX_3"
+    ]
```

Comment:
I don't think the exclusion change was called out in QPPA-6774 for this measure. Just making sure this change was expected.

Reply:
The ticket implies that this exclusion update should be for the new measure `PI_PPHI_2`

Reply:
resolved